### DISCUSSION TITLE
[suggestion] customizable length of presence

### DISCUSSION THREAD

**Discussion by DedaDev** [Jagailo/YetAnotherKeyDisplayer#28] [Ideas]
Letters for my use case dissapears too fast, and i cant change that.

**Comment by Jagailo** [comment]
Hello Deda,
Are you using the latest version?
Can you provide a gif/video of the problem?

**Comment by DedaDev** [comment]
It's not a problem, its a suggestion for a feature,
I want to set manually how long should letters stay in the box, in ms, not "on key up" as it is right now.

**Comment by Jagailo** [comment]
I don't think this is easy to do with the current workflow.
But if someone wish to do that, feel free to create a pull request.

**Comment by Jagailo** [comment]
The delay was added a couple of versions ago.
